pkg/daemon/ceph/client/upgrade.go

```diff
@@ -104,23 +104,32 @@ func GetAllCephDaemonVersions(context *clusterd.Context, clusterName string) (*C
 }
 
 // EnableMessenger2 enable the messenger 2 protocol on Nautilus clusters
-func EnableMessenger2(context *clusterd.Context) error {
-	_, err := context.Executor.ExecuteCommandWithOutput(false, "", "ceph", "mon", "enable-msgr2")
+func EnableMessenger2(context *clusterd.Context, clusterName string) error {
+	args := []string{"mon", "enable-msgr2"}
+	command, args := FinalizeCephCommandArgs("ceph", args, context.ConfigDir, clusterName)
```

Comment:
To run a `ceph` command, what about calling this helper? `NewCephCommand(context, clusterName, args).Run()`. Then I don't think you need to call `FinalizeCephCommandArgs()`.

Reply:
I was just trying to stay consistent with the content of the file. I can do that but then I'll do it for all the commands, sounds good?

Reply:
sounds good to make all the commands consistent this way.